apps/api/src/server/applications/application/project-updates/project-updates.mapper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param {import('@prisma/client').ProjectUpdate} projectUpdate
+ * @returns {import('@pins/applications').ProjectUpdate}
+ */
+export function mapProjectUpdate(projectUpdate) {
+	return {
+		id: projectUpdate.id,
```

Comment:
Should the get return all the fields such as title, sentToSubscribers, welsh...

Reply:
Yeah, makes sense to let the consumer decide what they want: https://github.com/Planning-Inspectorate/back-office/pull/992/commits/ef8c4dcea4eafc27a7e0c57c77cf18df806970be